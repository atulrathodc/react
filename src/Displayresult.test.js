import React from 'react';
import Link from './Link.react';
import renderer from 'react-test-renderer';
import Displatresult from './Displayresult';

test('snapshot test for temperatureInput component', () => {

    const component = renderer.create(
        <Displatresult all={[]}>Facebook</Displatresult>,
      );
      let comp = component.toJSON();

  expect(comp).toMatchSnapshot();


});