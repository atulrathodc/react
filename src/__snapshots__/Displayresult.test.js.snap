// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for temperatureInput component 1`] = `
<table
  cellSpacing="40"
>
  <thead>
    <tr>
      <th>
        Input Numerical Value
      </th>
      <th>
        Input Unit of Measure
      </th>
      <th>
        Target Unit of Measure
      </th>
      <th>
        Student Response
      </th>
      <th>
        Output
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
